### PR TITLE
Removing dayjs use and replacing it with TS Date object

### LINKED ISSUE #216: Remove dayjs library
Given react-big-calendar doesn't support dayjs and we have date-fns let's remove dayjs so we have a single date utility library. 
This issue involves `npm uninstall dayjs` and replacing any usage of dayjs with date-fns.

### PR DESCRIPTION
**Pull request by szeckirjr**
## Overview

Closes #216 

Changed the methods from dayjs for methods that do the same thing using TypeScript’s Date Object.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -130,16 +130,15 @@
   },
   "devDependencies": {
     "@storybook/addon-actions": "^6.1.21",
-    "@storybook/addon-essentials": "^6.1.21",
+    "@storybook/addon-essentials": "^6.2.9",
     "@storybook/addon-links": "^6.1.21",
     "@storybook/node-logger": "^6.1.21",
     "@storybook/preset-create-react-app": "^3.1.7",
-    "@storybook/react": "^6.1.21",
+    "@storybook/react": "^6.2.9",
```
Comment (aomi) [inline package.json:137]: How did this update? We should try and avoid updating packages unrelated to the task (and instead perhaps update these packages related to `storybook` together)

Comment (szeckirjr) [reply package.json:137]: sorry, that's totally my fault! should I change it back and commit again?

Comment (aomi) [reply package.json:137]: Yeah just revert those two bumps and push a new commit :+1: You can likely do:
```
npm install @storybook/addon-essential@6.1.21 storybook/react@6.1.21
```

**Review by aomi — APPROVED** [review]
Good stuff! Let's wait on merging this till the first meeting where we can use this to demo the deployment process!

**Pull request merged**